Modules/xxlimited_35.c

```diff
@@ -236,52 +236,58 @@ static PyMethodDef xx_methods[] = {
 PyDoc_STRVAR(module_doc,
 "This is a module for testing limited API from Python 3.5.");
 
+static PyObject *
+create_and_add_type(PyObject *module, const char *name, PyType_Spec *spec)
+{
+    PyObject *type = PyType_FromSpec(spec);
+    if (type == NULL) {
+        return NULL;
+    }
+    int rc = PyModule_AddObject(module, name, type);
+    if (rc < 0) {
```

Comment:
```suggestion
    if (PyModule_AddObject(module, name, type) < 0) {
```

I prefer not to use `rc` variable.

Reply:
No reason anymore; leftovers from first iteration of changes :)